kyc/kyc-sharing.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Share Gnosis Pay KYC with GP Partners
+description: This guide explains how to share existing Gnosis Pay KYC data with GP Partners using Sumsub's reusable token system.
+---
+
+## KYC Sharing Partners
+
+The following partners are currently available for KYC data sharing:
```

Comment:
I think we can start by giving a quick overview on what is KYC Sharing, mentioning this is only for Sumsub, and that there's a contract we sign between GP, sumsub, and the destination of the shared KYC data. 

we should also mention that since every company has a different kyc process, some GP approved kycs may be rejected at the destination

another thing to note is that we only support GP KYC -> Destination and not the other way around   

we should also link to sumsub's docs on this: https://docs.sumsub.com/docs/reusable-kyc